### DISCUSSION TITLE
[Scam Alert] treenvest project (پروژه سرمایه گذاری تری ان وست)

### DISCUSSION THREAD

**Discussion by micro73** [Blockunity/WarningList#4] [General]
تحلیل و بررسی پروژه  treenvest

1. سایت پروژه به ادرس blog.treenvest.com  - treenvest.com :
       در مورد نحوه فعالیت توضیحی ارائه نشده – مدیران سایت و پروژه به هیچ عنوان مشخص نیستند- دفترمرکزی و یا ادرسی در سایت موجود نیست – پلن درامدی در سایت موجود نیست – قسمت ثبت نام ندارد و فقط ورود دارد – سوشیال مدیا ندارد – فقط به اخبار حوزه کریپتو پرداخته است. 
جهت ثبت نام حتما باید لینک رفرال شخصی که قبلا ثبت نام کرده باشد را داشته باشید.
بعد از ثبت نام لینک برای اعضا ارسال میشه و میتونن حسابشون رو مدیریت کنند.
ایمیل : info@treenvest.com
2. توکن 3co مربوط به پروژه 
توکن در تاریخ Nov-15-2021 راه اندازی شده است. 
          ادرس کانترکت توکن : 0x1DE64424B63ebAEc3a29108cb9965796ef4deB3c
         سایت توکن به ادرس 3co.net  
      در کوین مارکت کپ هنوز لیست نشده، قیمت و مارکت کپ ندارد-، در دکس هنوز تامین نقدینگی هم نشده – صرفا فرد  سازنده توکن های تولید شده را در ولتی دارد و به گفته سازنده توکن قیمتش 1 سنت است در صورتی که عرضه و تقاضا باید قیمت را مشخص کند  و شما به عنوان سود حاصل از سرمایه گزاری treenvest میتوانید این ارز را برداشت کنید.... در حال حاضر حدود 49 درصد توکن ها در دو ولت هست.   سوشیال مدیا ندارد و فرد سازنده مشخص نیست. توکن ها غیرقابل فروش هست.
*در این مرحله به کد قراردادهوشمند این توکن میرسیم و چندتا از نکات خیلی جالب رو به صورت خلاصه برای شما باز میکنیم .
در مرحله اول اینکه به خودشون زحمت تغییر اسم کد های قراردادهوشمند توکن رو ندادند و به اسم cointoken ‌است که مشخص است که کد کپی شده است.
1 .   کانترکت توکن قابلیت توقف شدن رو داره ! گزینه unpause فقط توسط مالک کانترکت قابلیت اجرا دارد .
2.    کانترکت توکن قابلیت لیست سیاه داره!
3.    کانترکت توکن قابلیت Burn from wallet رو داره. یعنی توکن رو از داخل والت هرکسی میتونن بسوزونن
- بررسی ماشینی و خودکار توکن و مشکلاتش از لینک زیر قابل مشاهدس: 
https://bscheck.eu/bsc/0x1DE64424B63ebAEc3a29108cb9965796ef4deB3c
3. ایرادات هر دو سایت : 
ساپورت ندارد- ادرس ندارد - قوانین ندارد -  قالب وبسایت اماده خریداری شده - از ابزارهای آماده کانال های خبری برای پرکردن صفحات وب سایت استفاده شده است.
-	در اخرین اطلاعات دریافتی اعضای شبکه یا پرزنتور ها با دادن وعده صفته، چک تضمین و یا به نام زدن زمین معادل پول در حال ایجاد اعتماد سازی هستند 
_______________________________________________________________________________________________________________________________________

بررسی ادرس سایت در scamadviser.com
![1](https://user-images.githubusercontent.com/103103225/176452183-5dc3696f-42d9-413a-bdaf-ff0093b667c6.png)

مشکلات سایت blog.treenvest.com دریافت نمره 44 از 100
- صاحب وب سایت از سرویسی برای پنهان کردن هویت خود در WHOIS استفاده می کند
- طبق گفته Tranco این سایت رتبه الکسا پایینی دارد
- این سایت یک وب سایت در یک وب سایت دیگر است
- سعی کردیم محتوای سایت را تحلیل کنیم اما موفق نشدیم
- دامنه اصلی امتیاز اعتماد کمتری دارد
- به نظر می رسد این وب سایت توسط شرکتی با شهرت بد میزبانی می شود.
- این وب سایت در 30 روز گذشته توسط DNSFilter به عنوان تهدید علامت گذاری شده است

_______________________________________________________________________________________________________________________________________
** توضیحات ارائه شده توسط پرزنتور :::
برای ثبت نام برای هر فرد لینکی میفرستند که شامل کد دعوت نفربالایی است و میتواند ثبت نام کند.
طبق گفته پرزنتور با پول هایی که همه وارد میکنند دارند ترید میکنند و از اوون ترید سود به بقیه میدهند و لایو ترید برگزار میکنند که هویت هیچ کدام افراد مشخص نیست .
انتخاب پلن 6 ماهه، 5 درصد ماهانه پاداش میدهد ولی همه پاداش ها اخر 6 ماه قابل برداشت است.
انتخاب پلن 1 ساله ، 7 در صد ماهانه پاداش میدهد و ماهانه قابل برداشت است.
انتخاب پلن 2 ساله، 10 درصد ماهانه پاداش میدهد و ماهانه قابل برداشت است.
در پلن 1 ساله و 2 ساله میتوان اصل پول را از 2 ماه تا 9 ماه از ثبت جایگاه که گذشت برداشت کرد.
با ثبت هر شخص سه دست میتواند بسازد که هر دست میتواند شامل بی نهایت فرد باشد.
90 روز از ثبت جایگاه هر فرد فرصت دارد با زدن 5000 هزاردلار در هردست 1000 دلار پاداش دریافت کند.
6 ماه از ثبت جایگاه هر فرد فرصت دارد تا سه تا 100000 در هر دست بزند و 30000 هزار دلار پاداش دریافت کند با زدن این قسمت سطح 4 تا 20 برایش فعال میشود و از هر سطح 0.5 0.4 0.3 تا لول 20 پاداش میدهد
در ضمن 70 درصد سود را میتوانند ارز 3co دریافت کنند.

سطح 1 که از هر دست 1% پاداش میدهد
سطح 2 که از هر دست 0.5% پاداش میدهد
سطح 3 که از هر دست 5% پاداش میدهد
___________________________________________________________________________________________________________________________________
ادامه تحقیقات : 
در این پلتفرم امکان تهیه زیرمجموعه فراهم شده و برای معرفی شخص جدید به پلتفرم در نظر گرفته شده است.
این شرکت ادعای دروغین خودش در رابطه با ترید رو به این صورت به کاربران خودش ارائه میده .
لیست ترید ها به صورت هفتگی آپدیت میشه !!یعنی چی ؟ یعنی امروز میاد میگه من فلان ارز رو این قیمت خریدم و روی این قیمت فروختم  و دقیقا چیزی رو مینویسه اونجا که اتفاق افتاده باشه .و کاربر احساس سودی می کند که از طریق ترید دریافت می کند در صورتی که این سیستم به صورت هرمی مدیریت می شود و افراد قدیمی از پول افراد جدید الورود پرداخت می شود.
برنامه هایی که در متن بالا و پلن های شرکت مشاهده می کنید هرکسی برای ورود به سرمایه گذاری حداقل 6ماهه است و تا 6 ماه به هیچ عنوان به سرمایه خودش دسترسی نخواهد داشت و تمامی پول های واریزی تبدیل به توکن این شرکت میشه .

جمع واریزی ها به یک والت تقریباً 7 میلیون 600 هزار دلار در کمتر از 5 ماه !  
آدرس والت:
TTGaRkyhcnGhmLr4RDoQq8kKNfLrS57orC

**Comment by Blockunity** [comment]
update soon

**Comment by Blockunity** [comment]
جهت تکمیل اطلاعاتی نسبت به Treeinvest 
رفتار شباهت زیادی به یونیک فایننس داره.
با لانچ شدن  توکن UNQT شدت کلاهبرداری افزایش پیدا کرد.
توکن 3CO توسط این شرکت لانچ شده
که ادرس کانترکت این توکن برگرفته از توکن اسکم دیگری هست.
داخل بایننس اسمارت چین کانترت چک شده و فانکشن های خطرناکی یافت شده اعم از فانکشن های سوزاندن توکن ، مینت توکن و داخل لیست سیاه قرار دادن ادرس ها.

![IMG_20221113_130155_788.jpg](https://user-images.githubusercontent.com/72666334/201520776-17d49b29-f48e-4b2a-92c8-0c7df37e7dcf.jpg)

![IMG_20221113_130157_932.jpg](https://user-images.githubusercontent.com/72666334/201520784-b85b0982-3063-4e9b-9895-d57e9dc403ad.jpg)
طبق دیتای خود سایت کانترکت شباهتی هست
ادرس سایت :
https://bscscan.com/address/0x1de64424b63ebaec3a29108cb9965796ef4deb3c#code

در ادامه فانکشن های خطرناک ذکر شده:


	mapping(address => bool) tokenBlacklist;
	**event Blacklist(address indexed blackListed, bool value);**


  function transfer(address _to, uint256 _value) public returns (bool) {
    **require(tokenBlacklist[msg.sender] == false);**
    require(_to != address(0));
    require(_value <= balances[msg.sender]);

------------------------------------------------------
**contract PausableToken is StandardToken, Pausable {

  function transfer(address _to, uint256 _value) public whenNotPaused returns (bool) {
    return super.transfer(_to, _value);
  }**
-------------------------------------------------------

    }
	
	**function burn(uint256 _value) public {**
		_burn(msg.sender, _value);
	}

	function _burn(address _who, uint256 _value) internal {
		require(_value <= balances[_who]);
		balances[_who] = balances[_who].sub(_value);
		totalSupply = totalSupply.sub(_value);
		emit Burn(_who, _value);
		emit Transfer(_who, address(0), _value);
	}

    **function mint(address account, uint256 amount) onlyOwner public {**

        totalSupply = totalSupply.add(amount);
        balances[account] = balances[account].add(amount);
        emit Mint(address(0), account, amount);
        emit Transfer(address(0), account, amount);
    }

    
}

توکن در دسته اسکم با هشدار بالا قرار دارد.
